src/Fantomas.Tests/FormatConfigEditorConfigurationFileTests.fs

```diff
@@ -202,3 +202,24 @@ let ``print default editorconfig settings`` () =
     FormatConfig.Default
     |> EditorConfig.configToEditorConfig
     |> printfn "%s"
+
+[<Test>]
```

Comment:
Could you also add a test with only `fsharp_max_array_or_list_width` as single option, just to prove that the existing behavior still works as expected and does not need the `fsharp_array_or_list_multiline_formatter` setting.

Reply:
Added.

Reply:
I can't seem to find this, did you added it somewhere else maybe?

Reply:
Sorry, I misunderstood your meaning and had added it as a test in `NumberOfItemsTests.fs` for something different.  I have added the test now.